charts/jenkins/templates/jenkins-controller-statefulset.yaml

```diff
@@ -288,6 +288,20 @@ spec:
               mountPath: /run/secrets/additional
               readOnly: true
             {{- end }}
+            {{- if or .Values.controller.userContents .Values.controller.binaryUserContents }}
+            {{- range $content := .Values.controller.userContents }}
+            - name: jenkins-user-contents
+              mountPath: {{ $.Values.controller.jenkinsHome }}/userContent/{{ $content.name }}
+              subPath: {{ $content.name }}
+              readOnly: true
+            {{- end }}
+            {{- range $content := .Values.controller.binaryUserContents }}
+            - name: jenkins-user-contents
+              mountPath: {{ $.Values.controller.jenkinsHome }}/userContent/{{ $content.name }}
+              subPath: {{ $content.name }}
+              readOnly: true
+            {{- end }}
+            {{- end }}
```

Comment:
Is there a reason why you are mounting files individually instead of mounting all at once?

```
      - name: jenkins-user-contents
        mountPath: {{ $.Values.controller.jenkinsHome }}/userContent/
        readOnly: true
```

Reply:
If you mount the Configmap, K8S will create symlinks `foobar.txt -> ..data/foobar.txt` where `..data -> ..2020_03_03_16_40_36.675612011`

Jenkins seems to be not supporting symlinks at this point - neither by displaying nor serving it:
![image](https://user-images.githubusercontent.com/508684/183065166-d19f9c48-796f-4e72-bdeb-df394a3d3f71.png)
Might be related to [Jenkins Security Advisory 2021-01-13](https://www.jenkins.io/security/advisory/2021-01-13/)?

So you can not access `userContent/foobar.txt` directly but would need to go via something like `userContent/..2020_03_03_16_40_36.675612011/foobar.txt` - and this is an always changing URL.

Reply:
Thx for the clarification.